src/lib/storage/fake_device_factory.rb

```diff
@@ -86,8 +84,9 @@ def initialize(devicegraph)
         super(devicegraph)
         @partitions     = {}
         @disks          = Set.new
-        @first_free_cyl = {}
-        @cyl_count      = {}
+        @free_blob      = nil
+        @free_regions   = []
+        @mbr_gap        = nil
```

Comment:
That is, I want to have the internal class vars shown in one place without having to add some dummy initialization.